### DISCUSSION TITLE
Hamlib 4.6

### DISCUSSION THREAD

**Discussion by N0NB** [Hamlib/Hamlib#1641] [Announcements]
### Version 4.6 Release of Hamlib and included utilities.

Along with the GNU Autotools generated source archive, `hamlib-4.6.tar.gz`, binary packages for Windows 32 and 64 bit systems are included as both a ZIP file or self extracting EXE file.

All files are signed by N0NB's gpg key for additional assurance.

**Note:**  The files named `Source code (zip)` and `Source code (tar.gz)` are generated by GitHub and require a complete installation of the GNU Autotools to bootstrap the build system.

**Full Changelog**: https://github.com/Hamlib/Hamlib/compare/4.5.5...4.6

<hr /><em>This discussion was created from the release <a href='https://github.com/Hamlib/Hamlib/releases/tag/4.6'>Hamlib 4.6</a>.</em>

**Comment by vk1kcm** [comment]
Since the "rig_caps" change broke a lot of clients and it's been rolled back for 4.6.1, is there a timeline for releasing 4.6.1?  We can't package up 4.6 due to the breakage so I imagine most packaging systems will stick with 4.5.5 until 4.6.1 comes out.

**Comment by mdblack98** [comment]
In the next few days

 

    On Monday, January 20, 2025 at 03:20:52 AM CST, Carl Makin ***@***.***> wrote:  
 
 


Since the "rig_caps" change broke a lot of clients and it's been rolled back for 4.6.1, is there a timeline for releasing 4.6.1? We can't package up 4.6 due to the breakage so I imagine most packaging systems will stick with 4.5.5 until 4.6.1 comes out.

**Comment by z34pec** [comment]
Would be there any DEB packages for ubuntu / debian OS ?

**Comment by N0NB** [comment]
Debian packages are built by Debian or Ubuntu maintainers.  For Debian 4.6.2 is in Unstable and Testing.  It has not been added to the Backports archive and so is not available in Stable Backports.
